<<<<<<< HEAD
﻿2024-11-26 10:25:22,398 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 10:25:22,401 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
2024-11-26 10:25:22,865 - _internal.py[line:97] - INFO:  * Detected change in 'E:\\github_repositories\\Emotion-Analysis-Project\\Emotion-Analysis\\weibo_crawler\\weibo_crawler\\settings.py', reloading
2024-11-26 10:26:42,382 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:26:42,382 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:26:42,383 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:26:42,383 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:26:42,420 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:26:42,421 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:26:42,422 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:26:42,423 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:26:42,424 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:26:42,425 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:26:42,426 - main.py[line:53] - ERROR: 无法删除 weibo_crawler\weibo_data.csv：[WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'weibo_crawler\\weibo_data.csv'
2024-11-26 10:26:42,427 - main.py[line:53] - ERROR: 无法删除 weibo_crawler\weibo_data.csv：[WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'weibo_crawler\\weibo_data.csv'
2024-11-26 10:26:42,427 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\pie_chart.png
2024-11-26 10:26:42,428 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\pie_chart.png
2024-11-26 10:26:42,429 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_all.png
2024-11-26 10:26:42,430 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_all.png
2024-11-26 10:26:42,431 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 10:26:42,432 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 10:26:42,432 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
2024-11-26 10:26:42,433 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
2024-11-26 10:26:43,401 - _internal.py[line:97] - INFO:  * Detected change in 'E:\\github_repositories\\Emotion-Analysis-Project\\Emotion-Analysis\\weibo_crawler\\weibo_crawler\\settings.py', reloading
2024-11-26 10:29:19,562 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:29:19,562 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:29:19,564 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:29:19,564 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:29:19,565 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:29:19,566 - main.py[line:44] - INFO: 已删除文件：weibo_crawler\weibo_data.csv
2024-11-26 10:29:19,566 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\pie_chart.png
2024-11-26 10:29:19,567 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_all.png
2024-11-26 10:29:19,567 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 10:29:19,568 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
2024-11-26 10:29:19,587 - _internal.py[line:97] - INFO:  * Detected change in 'E:\\github_repositories\\Emotion-Analysis-Project\\Emotion-Analysis\\weibo_crawler\\weibo_crawler\\settings.py', reloading
2024-11-26 10:32:04,687 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:32:04,688 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:32:04,688 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:32:04,689 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:32:04,689 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:32:04,690 - main.py[line:44] - INFO: 已删除文件：weibo_crawler\weibo_data.csv
2024-11-26 10:32:04,691 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\pie_chart.png
2024-11-26 10:32:04,691 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_all.png
2024-11-26 10:32:04,692 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 10:32:04,692 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
2024-11-26 10:32:05,432 - _internal.py[line:97] - INFO:  * Detected change in 'E:\\github_repositories\\Emotion-Analysis-Project\\Emotion-Analysis\\weibo_crawler\\weibo_crawler\\settings.py', reloading
2024-11-26 10:41:53,545 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:41:53,546 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:41:53,547 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:41:53,547 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:41:53,548 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:41:53,548 - main.py[line:44] - INFO: 已删除文件：weibo_crawler\weibo_data.csv
2024-11-26 10:41:53,549 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\pie_chart.png
2024-11-26 10:41:53,549 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_all.png
2024-11-26 10:41:53,550 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 10:41:53,550 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
2024-11-26 10:41:54,190 - _internal.py[line:97] - INFO:  * Detected change in 'E:\\github_repositories\\Emotion-Analysis-Project\\Emotion-Analysis\\weibo_crawler\\weibo_crawler\\settings.py', reloading
2024-11-26 10:41:58,456 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:41:58,457 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:41:58,457 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:41:58,458 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:41:58,458 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:41:58,459 - main.py[line:53] - ERROR: 无法删除 weibo_crawler\weibo_data.csv：[WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'weibo_crawler\\weibo_data.csv'
2024-11-26 10:41:58,459 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\pie_chart.png
2024-11-26 10:41:58,460 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_all.png
2024-11-26 10:41:58,460 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_positive.png
2024-11-26 10:41:58,461 - main.py[line:36] - ERROR: 文件路径不存在：static\wordclouds\wordcloud_negative.png
2024-11-26 10:41:59,522 - _internal.py[line:97] - INFO:  * Detected change in 'E:\\github_repositories\\Emotion-Analysis-Project\\Emotion-Analysis\\weibo_crawler\\weibo_crawler\\settings.py', reloading
2024-11-26 10:42:55,618 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:42:55,618 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\all_words.csv
2024-11-26 10:42:55,619 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:42:55,619 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\predicted_results.csv
2024-11-26 10:42:55,620 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:42:55,620 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\positive_words.csv
2024-11-26 10:42:55,621 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:42:55,621 - main.py[line:36] - ERROR: 文件路径不存在：data_visualization\negative_words.csv
2024-11-26 10:42:55,621 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:42:55,622 - main.py[line:36] - ERROR: 文件路径不存在：model\processed_data.csv
2024-11-26 10:42:55,623 - main.py[line:53] - ERROR: 无法删除 weibo_crawler\weibo_data.csv：[WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'weibo_crawler\\weibo_data.csv'
2024-11-26 10:42:55,623 - main.py[line:53] - ERROR: 无法删除 weibo_crawler\weibo_data.csv：[WinError 32] 另一个程序正在使用此文件，进程无法访问。: 'weibo_crawler\\weibo_data.csv'
=======
﻿2024-11-25 18:02:14,071 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:02:14] "[36mGET /static/js/gptChat.js HTTP/1.1[0m" 304 -
2024-11-25 18:02:14,072 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:02:14] "[36mGET /static/css/fontawesome-all.min.css HTTP/1.1[0m" 304 -
2024-11-25 18:02:14,102 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:02:14] "[36mGET /static/images/overlay.png HTTP/1.1[0m" 304 -
2024-11-25 18:02:14,104 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:02:14] "[36mGET /static/images/bg.jpg HTTP/1.1[0m" 304 -
2024-11-25 18:02:14,105 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:02:14] "[36mGET /static/webfonts/fa-regular-400.woff2 HTTP/1.1[0m" 304 -
2024-11-25 18:02:27,455 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:02:27] "POST /page/gpt_suggestion HTTP/1.1" 200 -
2024-11-25 18:03:23,533 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "GET / HTTP/1.1" 200 -
2024-11-25 18:03:23,591 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,604 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/images/pic04.jpg HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,608 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/js/jquery.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,615 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/js/browser.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,621 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/js/breakpoints.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,624 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/js/util.js HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,627 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,633 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/css/fontawesome-all.min.css HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,639 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/js/analysis.js HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,642 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/js/gptChat.js HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,665 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/images/overlay.png HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,670 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/images/bg.jpg HTTP/1.1[0m" 304 -
2024-11-25 18:03:23,673 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:03:23] "[36mGET /static/webfonts/fa-regular-400.woff2 HTTP/1.1[0m" 304 -
2024-11-25 18:04:16,316 - __init__.py[line:113] - DEBUG: Building prefix dict from the default dictionary ...
2024-11-25 18:04:16,317 - __init__.py[line:132] - DEBUG: Loading model from cache C:\Users\Vert\AppData\Local\Temp\jieba.cache
2024-11-25 18:04:17,711 - __init__.py[line:164] - DEBUG: Loading model cost 1.394 seconds.
2024-11-25 18:04:17,711 - __init__.py[line:166] - DEBUG: Prefix dict has been built successfully.
2024-11-25 18:06:45,518 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "GET / HTTP/1.1" 200 -
2024-11-25 18:06:45,566 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/css/main.css HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,567 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/images/pic04.jpg HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,568 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/js/jquery.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,585 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/js/browser.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,590 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/js/util.js HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,591 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/js/breakpoints.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,591 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,592 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/js/analysis.js HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,599 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/css/fontawesome-all.min.css HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,603 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/js/gptChat.js HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,634 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/images/overlay.png HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,635 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/images/bg.jpg HTTP/1.1[0m" 304 -
2024-11-25 18:06:45,640 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:06:45] "[36mGET /static/webfonts/fa-regular-400.woff2 HTTP/1.1[0m" 304 -
2024-11-25 18:22:59,319 - __init__.py[line:113] - DEBUG: Building prefix dict from the default dictionary ...
2024-11-25 18:22:59,326 - __init__.py[line:132] - DEBUG: Loading model from cache C:\Users\Vert\AppData\Local\Temp\jieba.cache
2024-11-25 18:22:59,941 - __init__.py[line:164] - DEBUG: Loading model cost 0.614 seconds.
2024-11-25 18:22:59,941 - __init__.py[line:166] - DEBUG: Prefix dict has been built successfully.
2024-11-25 18:24:11,235 - main.py[line:62] - INFO: Emotion analysis completed successfully
2024-11-25 18:24:11,236 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:24:11] "POST /page/spider_analysis/topic HTTP/1.1" 200 -
2024-11-25 18:24:11,257 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:24:11] "GET /static/wordclouds/wordcloud_all.png HTTP/1.1" 200 -
2024-11-25 18:24:11,258 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:24:11] "GET /static/wordclouds/wordcloud_negative.png HTTP/1.1" 200 -
2024-11-25 18:24:11,259 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:24:11] "GET /static/wordclouds/wordcloud_positive.png HTTP/1.1" 200 -
2024-11-25 18:28:37,224 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "GET / HTTP/1.1" 200 -
2024-11-25 18:28:37,274 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "GET /static/css/main.css HTTP/1.1" 200 -
2024-11-25 18:28:37,283 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/images/pic04.jpg HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,284 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/js/jquery.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,292 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/js/browser.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,293 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/js/breakpoints.min.js HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,294 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/js/util.js HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,295 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,298 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/css/fontawesome-all.min.css HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,300 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/js/analysis.js HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,301 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/js/gptChat.js HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,319 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/images/overlay.png HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,320 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/images/bg.jpg HTTP/1.1[0m" 304 -
2024-11-25 18:28:37,321 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:37] "[36mGET /static/webfonts/fa-regular-400.woff2 HTTP/1.1[0m" 304 -
2024-11-25 18:28:43,182 - __init__.py[line:113] - DEBUG: Building prefix dict from the default dictionary ...
2024-11-25 18:28:43,183 - __init__.py[line:132] - DEBUG: Loading model from cache C:\Users\Vert\AppData\Local\Temp\jieba.cache
2024-11-25 18:28:44,629 - __init__.py[line:164] - DEBUG: Loading model cost 1.447 seconds.
2024-11-25 18:28:44,630 - __init__.py[line:166] - DEBUG: Prefix dict has been built successfully.
2024-11-25 18:28:54,156 - main.py[line:24] - ERROR: Error in weibo_crawler: [WinError 2] 系统找不到指定的文件。: 'C:\\Users\\Vert\\OneDrive\\Desktop\\team\\Emotion-Analysis-Project\\Emotion-Analysis\\model\\weibo_crawler'
2024-11-25 18:28:54,157 - _internal.py[line:97] - INFO: 127.0.0.1 - - [25/Nov/2024 18:28:54] "POST /page/spider_analysis/topic HTTP/1.1" 200 -
2024-11-25 18:30:44,017 - __init__.py[line:113] - DEBUG: Building prefix dict from the default dictionary ...
2024-11-25 18:30:44,027 - __init__.py[line:132] - DEBUG: Loading model from cache C:\Users\Vert\AppData\Local\Temp\jieba.cache
2024-11-25 18:30:45,854 - __init__.py[line:164] - DEBUG: Loading model cost 1.828 seconds.
2024-11-25 18:30:45,855 - __init__.py[line:166] - DEBUG: Prefix dict has been built successfully.
2024-11-25 18:33:05,803 - main.py[line:62] - INFO: Emotion analysis completed successfully
>>>>>>> 4a312c24d8f68bd43806bfdb4943ca0aab106dda
